codalab/worker/rest_client.py

```diff
@@ -114,7 +114,7 @@ def _make_request(
                     raise RestClientException('Invalid JSON: ' + response_data, False)
 
     def _upload_with_chunked_encoding(
-        self, method, url, query_params, fileobj, progress_callback=None
+        self, method, url, query_params, fileobj, pass_self=False, progress_callback=None
```

Comment:
What does pass_self mean? Can you document here and in the other function where you add it

Reply:
Done.